lib/preview_web/live/preview_live.ex

```diff
@@ -58,7 +62,7 @@ defmodule PreviewWeb.PreviewLive do
 
     {:noreply,
      push_patch(socket,
-       to: Routes.preview_path(socket, :index, package, version, filename),
+       to: Routes.preview_path(socket, :index, package, version) <> "/show/#{filename}",
```

Comment:
We don't get a path helper for this route?

Reply:
We do but it accepts a list of strings (to join with /) and i already have a full path. Happy to change of course

Reply:
Okay, this looks good in that case.